lib/auth/index.js

```diff
@@ -67,7 +68,17 @@ module.exports = settings => {
       })
       .then(() => next())
       .catch(next);
+  };
+
+  router.use(setUser);
 
+  router.use((req, res, next) => {
+    req.user.refreshProfile = (req, res, next) => {
+      req.session.profile.expiresAt = Date.now();
+      return setUser(req, res, next);
+    };
```

Comment:
it would be cleaner to define this method on the user object above - after `can` and `allowedActions`